src/Audit.php

```diff
@@ -179,4 +179,80 @@ public function getModified($json = false, $options = 0, $depth = 512)
 
         return $json ? json_encode($modified, $options, $depth) : $modified;
     }
+
+    /**
+     * @return return all audits that were related to $this
+     */
+    public function getRelatedAudits()
+    {
+        if ($this->event == 'related') {
+            return;
+        }
+        $audit_class = Config::get('audit.implementation', \OwenIt\Auditing\Models\Audit::class);
+        $RelatedAuditObjArr = $audit_class::where('relation_id', '=', $this->relation_id)
+                                           ->where('event', '=', 'related')->get();
+
+        return $RelatedAuditObjArr;
+    }
+
+    /**
+     * Get the relating Audit.
+     *
+     * @return mixed
+     */
+    public function getRelatingAudit()
+    {
+        $audit_class = Config::get('audit.implementation', \OwenIt\Auditing\Models\Audit::class);
+        if ($this->event !== 'related') {
+            return;
+        }
+        /** @var \OwenIt\Auditing\Models\Audit $RelatingAuditObj */
+        $RelatingAuditObj = $audit_class::where('relation_id', '=', $this->relation_id)
+                                           ->where('event', '!=', 'related')->get()->first();
+        $RelatingAuditObj->setIsRelating(true);
+
+        return $RelatingAuditObj;
+    }
+
+    public $is_relating = false;
```

Comment:
Attributes go on top!

Reply:
OK, again if there is a list of conventions beyond continuous-integration/styleci/pr, please let me know and I will happily conform.

Reply:
Sorry, but this one is a no brainer. Again, look at the existing codebase.